bitrise.yml

```diff
@@ -2,173 +2,200 @@ format_version: 3
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 app:
   envs:
-  - GO_PROJECT_PATH: github.com/bitrise-io/bitrise-webhooks
-  - PORT: "4000"
-  - IS_USE_GIN: "yes"
-    opts:
-      description: |
-        If set to "yes" we'll use github.com/codegangsta/gin to live-reload the code on change.
-  - SEND_REQUEST_TO: $SEND_REQUEST_TO
-    opts:
-      description: |
-        Optionally, define this in your .bitrise.secrets.yml, to send requests to the specified URL.
-  - RELEASE_START_BRANCH: master
-  - RELEASE_DESTINATION_BRANCH: prod
+    - GO_PROJECT_PATH: github.com/bitrise-io/bitrise-webhooks
+    - PORT: "4000"
+    - IS_USE_GIN: "yes"
+      opts:
+        description: |
+          If set to "yes" we'll use github.com/codegangsta/gin to live-reload the code on change.
+      opts:
+        description: |
+          Optionally, define this in your .bitrise.secrets.yml, to send requests to the specified URL.
+    - RELEASE_START_BRANCH: master	
+    - RELEASE_DESTINATION_BRANCH: prod
 workflows:
   _install_test_tools:
     steps:
-    - script:
-        title: Install required testing tools
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
+      - script:
+          title: Install required testing tools
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
 
-            # Check for unhandled errors
-            go get -u github.com/kisielk/errcheck
+                # Check for unhandled errors
+                go get -u github.com/kisielk/errcheck
 
-            # Go lint
-            go get -u golang.org/x/lint/golint
-  create-release:
+                # Go lint
+                go get -u golang.org/x/lint/golint
+  publish-release-package:
     steps:
-    - ensure-clean-git: {}
-    - script:
-        title: bump version number, then merge into 'prod' branch, tag & commit &
-          push
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -e
+      - script:
+          title: Start the release flow
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -e
+                bitrise run configure-git-credentials
```

Comment:
Pls use chaining workflows instead of running bitrise inside bitrise :)
https://devcenter.bitrise.io/jp/bitrise-cli/workflows/#chaining-workflows-and-reusing-workflows